code/style.css

```diff
@@ -1,4 +1,222 @@
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 10px 60px 10px 20px;
+  font-family: 'Carter One', cursive;
+  color: #F2002A;
+}
+
+.logo-image{
+  height: 100%;
+}
+
+nav{
+  display: none;
+}
+
+a{
+  color: #F2002A;
+}
+
+p{
+  text-align: justify;
+  /* padding: 10px 0 0 0; */
+}
+ 
+/* following formating is affecting the big news section */
+.big-news{
+  padding: 40px 10px 40px 10px;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  justify-content: center;
+  border-bottom: 2px solid white;
+}
+
+.big-news-visual{
+  width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.big-news-picture{
+  width: 95%;
+  -webkit-transform: scale(1);
+	transform: scale(1);
+	-webkit-transition: .3s ease-in-out;
+	transition: .3s ease-in-out;
+  box-shadow: 0 0 0 8px white;
+  border-radius: 3px;
+}
+
+.big-news-picture:hover{
+  -webkit-transform: scale(1.02);
+	transform: scale(1.02);
+}
+.big-news-text{
+  /* width: 95%; */
+  padding: 8px;
+}
+
+/* following formating is affecting the small news sections */
+.small-news{
+  display: grid;
+  grid-template-columns: 1fr;
+  padding-top: 10px;
+  border-bottom: 3px solid white;
+}
+
+.small-news-item{
+  padding: 15px;
+  display: flex;
+  flex-direction: column;
+  gap: 5px;
+  font-size: 12px;
+  justify-content: center;
+}
+
+.small-news-picture-container{
+  width: 100%;
+  overflow: hidden;
+  box-sizing: border-box;
+}
+.small-news-picture{
+  width: 100%;
+  border-radius: 3px;
+  -webkit-transform: scale(1);
+	transform: scale(1);
+	-webkit-transition: .3s ease-in-out;
+	transition: .3s ease-in-out;
+  box-shadow: 0 0 0 8px white;
+}
+
+.small-news-picture:hover{
+  -webkit-transform: scale(1.03);
+	transform: scale(1.03);
+}
+
+.gap-line{
+  width: 100%;
+  height: 5px;
+  margin: 0 20% 0 20%;
+  background-color: white;
+}
+
+.bottom-footer{
+  display: flex;
+  flex-direction: row;
+  gap: 25px;
+  justify-content: center;
+  padding: 15px;
+}
+
+.footer-image{
+  width: 30px;
+  height: 30px;
+}
+
+.small-logo{
+  width: 100%;
+}
+/* ------i-PAD VERSION ------------*/
+@media (min-width: 768px){
+  /* all the changes that are affecting the header container and child elements */
+  .header{
+    justify-content: space-around;
+    height: 80px;
+  }
+  .logo-image{
+    display: inline-block;
+    height: 100%;
+  }
+
+  nav{
+    display: flex;
+    justify-content: space-around;
+    width: 100%;
+    position: relative;
+    top: -40px;
+    padding: 0 25px;
+  }
+  .title-text{
+    display: none;
+  }
+  
+  /* all the changes that affect the big-news container */
+  .big-news{
+    flex-direction: row;
+    align-items:flex-start;
+  }
+  .big-news-visual{
+    width: 50%;
+  }
+  .big-news-text{
+    width: 50%;
+  }
+
+  /* all the changes that affect the small-news contianer CSS GRID used */
+  .small-news{
+    grid-template-columns: 1fr 1fr;
+  }
+
 }
+/* ---------DESKTOP VERSION ---------*/
```

Comment:
The changes to the different media queries are very precise and well defined. It is all there what's needed but not more than necessary. Great job!